### DISCUSSION TITLE
Tutorial

### DISCUSSION THREAD

**Discussion by Hk669** [Hk669/Open-Source-Recommender#1] [Announcements]
## 👋 Welcome to the Open-Source-Recommender!

Welcome to the ultimate tool for discovering the best open-source projects tailored just for you! This tutorial will help you make the most out of our Open-Source-Recommender.

#### **1. Quick Recommendations**
If you’re eager to see what’s available right now, simply click the **`Get Recommendations`** button. This will fetch a list of projects based solely on the most popular GitHub repositories, ensuring you get a broad view of the top options out there.

#### **2. Personalize Your Experience**
For a more refined and personalized set of recommendations, you can tailor your search based on preferred languages and topics. Here’s how:

- **Preferred Languages**: 
  - Want recommendations that favor certain programming languages? Just enter the names of the languages you’re interested in. For instance, you can type `Python` or `JavaScript` and hit enter to add them to your list.
  
- **Relevant Topics**:
  - To further refine your recommendations, you can add specific topics related to your interests. For example, if you’re into artificial intelligence, you might add topics like `ai` or `agentic-ai`.

#### **3. How to Add Languages and Topics**
  - **Languages**: Type the name of the language and press enter to add it to your list. This helps us understand which languages you prefer for the recommendations.
  - **Topics**: Similarly, type your topics of interest (e.g., `machine learning`, `web development`) and press enter to include them. This will help filter the recommendations based on your interests.

### 🌟 Explore Your Recommendations

Once you’ve set your preferences, your personalized recommendations will be displayed. Whether you’re looking for cutting-edge AI projects or innovative web development tools, our recommender will find the best open-source projects for you.

### 🏆 Ready to Discover?

Head over to [gitmatch.in](https://gitmatch.in) to get started with your personalized recommendations and explore the best open-source projects that match your interests.

Enjoy discovering new projects and happy coding!

**Comment by Hk669** [comment]
i do burn a lot of oai credits and mainly my AWS credits :)

**Reply by shashiwadana** [reply]
Hi! I'm interested in this project. Could you please guide me on how to run this?
